client/my-sites/themes/thanks-modal.jsx

```diff
@@ -201,20 +200,15 @@ const ThanksModal = React.createClass( {
 
 export default connect(
 	( state, { site } ) => {
-		if ( ! site ) {
-			return {};
-		}
-
-		const siteIdOrWpcom = isJetpackSite( state, site.ID ) ? site.ID : 'wpcom';
-		const currentThemeId = getActiveTheme( state, site.ID );
-		const currentTheme = getTheme( state, siteIdOrWpcom, currentThemeId );
+		const currentThemeId = site && getActiveTheme( state, site.ID );
+		const currentTheme = currentThemeId && getTheme( state, site.ID, currentThemeId );
```

Comment:
(Mind the difference of `getTheme` -- which gets theme data for wpcom themes from the `wpcom` subtree -- and `getActiveTheme`, which only makes sense per-actual-siteId.)